IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -782,8 +782,8 @@ - (BOOL)saveAccessTokenWithConfiguration:(MSIDConfiguration *)configuration
     MSIDAccessToken *accessToken = [factory accessTokenFromResponse:response configuration:configuration];
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->